mitiq/shadows/shadows_utils.py

```diff
@@ -8,13 +8,74 @@
 # LICENSE file in the root directory of this source tree.
 
 """Defines utility functions for classical shadows protocol."""
-from typing import List, Tuple
+from typing import Tuple, List, Any
 
 import numpy as np
 from numpy.typing import NDArray
+import cirq
 
 import mitiq
 
+from itertools import product
+
+Paulis = [
+    cirq.I._unitary_(),
+    cirq.X._unitary_(),
+    cirq.Y._unitary_(),
+    cirq.Z._unitary_(),
+]
```

Comment:
Globals are usually all caps.
```suggestion
PAULIS = [
    cirq.I._unitary_(),
    cirq.X._unitary_(),
    cirq.Y._unitary_(),
    cirq.Z._unitary_(),
]
```